### PR TITLE
Fix NMEA message settings for AG3335

### PR DESCRIPTION
**Pull request by fifieldt**
In collaboration with the great @GPSFan, we present fixed NMEA message configuration for the AG3335.

"only RMC & GGA are used by Meshtastic", and added some delays to see if it assists.

### CODE REVIEW

**Comment by GPSFan** [comment]
This issue may need some hardware debugging, if anyone has an opened Seeed T1000E who could loan it for a while it might be useful.

File: src/gps/GPS.cpp [hunk 1]
```diff
@@ -507,18 +507,28 @@ bool GPS::setup()
...
+            delay(250); // GSV OFF
+            _serial_gps->write("$PAIR062,4,1*3B");
+            delay(250); // RMC ON
+            _serial_gps->write("$PAIR062,5,0*3B");
+            delay(250); // VTG OFF
+            _serial_gps->write("$PAIR062,6,0*38");
+            delay(250); // ZDA ON
+
+            _serial_gps->write("$PAIR080,1*2F");
+            delay(250); // Fitness mode!
```
Comment (caveman99) [inline src/gps/GPS.cpp:530]: I couldn't make these commands work on the T1000E either, but noticed the CR/LF is missing. (\r\n) Even with those they show no effect. Are we sure TX is even connected on this chip?

Comment (fifieldt) [reply src/gps/GPS.cpp:530]: See comments in gnss on discourse today. I can't get any commands working either :(

Comment (fifieldt) [reply src/gps/GPS.cpp:530]: The CRLF does need yo be fixed though :)

**Comment by caveman99** [comment]
Closing because of merge conflicts, and manually testing this it doesn't make a difference anyway.

**Pull request closed without merging**